npm.js

```diff
@@ -59,6 +67,10 @@ exports.translate = function(load){
 				if(pkg.browser){
 					delete pkg.browser.transform;
 				}
+				if(pkg.system && pkg.system.npmAlgorithm === "nested" && context.isFlatFileStructure === true) {
```

Comment:
I don't think we should warn here. npmAlgorithm is essentially a dev-mode option. If a library author is using `nested` because they themselves are developing with npm2, that doesn't mean it won't work with npm3.  Only the project that is running `npm install` matters.